README.md

```diff
@@ -1,24 +1,172 @@
-# README
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Screenshots](#screenshots)
+    - [Live Demo](#link-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-Things you may want to cover:
+# 📖 SpendLog <a name="about-project"></a>
 
-* Ruby version
+A web application for management of your expenses. The application groups related transactions under categories for ease of visualizing.
 
-* System dependencies
+## 🛠 Built With <a name="built-with"></a>
 
-* Configuration
+### Tech Stack <a name="tech-stack"></a>
 
-* Database creation
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML(erb), CSS</li>
+  </ul>
+</details>
 
-* Database initialization
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby-on-Rails</li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgress</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- Register and login
+- Category List (Home page)
+  - List all caegories for a user
+  - Add New Category, to create new category
+- Category detail page (Transactions)
+  - List all transactions under a category
+  - Add New Transaction, to create new transaction
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Screenshots <a name="screenshots"></a>
+
+<div align="center">
+  <img src="./app/assets/images/splash.png" alt="Splash" />
+  <img src="./app/assets/images/login.png" alt="Login" />
+  <img src="./app/assets/images/categories.png" alt="Category list" />
+  <img src="./app/assets/images/transactions.png" alt="transactions" />
+  <img src="./app/assets/images/new_transaction.png" alt="new transaction" />
+</div>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo <a name="live-demo"></a>
+
+[Budget Flow Live Demo](https://spendlog.onrender.com)
```

Comment:
- [ ] when trying to add a transaction in the live demo it breaks, please have a look at what is going wrong with it.
![image](https://github.com/adamilare/spendlog/assets/39240820/78b4eccc-943c-4d04-b8b8-4dde5ee94ea7)

Reply:
@mikethreels thank you for your review.
Can you please provide me more info of what led to the failure?
It seems there is an edge case I've not catch.

Thank you